src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--month-length--31">
```

Comment:
```suggestion
    <section class="calendar calendar--month-length-31 calendar--start-day-mon">
```

Reply:
add start day modifier to calendar block(use first-child to access to first cell)

Reply:
> 

Thank you for review!)
Modifier was replaced to calendar.

Could you please clarify this suggestion (value of modifier set after "-" not "--"?)

Reply:
Because 31 it is not modificator, it's only part of `--month-length` modificator